.github/workflows/build.yml

```
@@ -98,3 +98,23 @@ jobs:
       - name: Setup tmate debug session
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: ["1.18", "1.19"]
```

We prefer block-style over flow-style.

```suggestion
        go-version:
          - "1.18"
          - "1.19"
```

---

@mcdonnnj resolved with [7751db5](https://github.com/cisagov/con-pca-tasks/pull/1/commits/7751db53ea6145e573bea708aea853b28cc8aea0)